vernac/comFixpoint.ml

```diff
@@ -225,7 +225,8 @@ let interp_recursive ~program_mode ~cofix fixl notations =
   (* Instantiate evars and check all are resolved *)
   let sigma = solve_unif_constraints_with_heuristics env_rec sigma in
   let sigma, _ = nf_evars_and_universes sigma in
-  let fixdefs = List.map (fun c -> Option.map EConstr.(to_constr sigma) c) fixdefs in
+  (* XXX: We still have evars here in Program *)
```

Comment:
Yes, I guess we should rather have an interface change in obligations.mli so that it takes EConstr.t's there.